components/layout/navigation.tsx

```diff
@@ -27,13 +27,13 @@ export function Navigation() {
           className: "w-8 h-8",
           "aria-hidden": true,
         })}
-        <span className="text-xs">{item.name}</span>
+        <span className="text-xs text-center">{item.name}</span>
```

Comment:
@ihsanrabb Is this necessary? Didn't `items-center` above already sort the centering?

Reply:
I think it's better to add a `text-center`. `items-center` only center an icon on smaller screen @resir014 
![bottom-nav](https://user-images.githubusercontent.com/47909781/127001549-8ad30b3e-1686-4ca3-8006-b9612e0ad9bb.jpeg)

Reply:
Okay. Then I think we should move this up higher up the element tree, alongside `items-center justify-center` @ihsanrabb